terraform/private-law.tf

```
@@ -0,0 +1,19 @@
+locals {
+  pl_non_prod_domains = ["mail-prl-nonprod.aat.platform.hmcts.net", "mail-prl-nonprod.demo.platform.hmcts.net", "mail-prl-nonprod.ithc.platform.hmcts.net", "mail-prl-nonprod.perftest.platform.hmcts.net", "mail-prl-nonprod.preview.platform.hmcts.net"]
+  pl_prod_domains     = ["mail-prl.platform.hmcts.net"]
```

team config say this team uses prl abbreviation:
https://github.com/hmcts/cnp-jenkins-config/blob/500eae88868fc2bda906a42d3774109053f66f65/team-config.yml#L661-L669

```suggestion
  prl_prod_domains     = ["mail-prl.platform.hmcts.net"]
```